release-please-config.json

```diff
@@ -1,8 +1,6 @@
 {
   "bootstrap-sha": "3b7fca13b911d359016ad58693ccfc8f95d29b0c",
-  "plugins": [
-    "node-workspace"
-  ],
+  "plugins": ["node-workspace"],
```

Comment:
that's useless, as the script will re-add the new lines next time

Reply:
it was from prettier, should we remove this file from prettier review?

Reply:
yes